web/client/components/catalog/RecordItem.jsx

```diff
@@ -62,18 +87,81 @@ const RecordItem = React.createClass({
         }}/>);
 
     },
-    renderWMSButtons(wms) {
+    renderButtons(record) {
+        if (!record || !record.references) {
+            // we don't have a valid record so no buttons to add
+            return null;
+        }
+        // let's extract the references we need
+        let wms = head(record.references.filter(reference => reference.type && (reference.type === "OGC:WMS"
+            || ((reference.type.indexOf("OGC:WMS") > -1 && reference.type.indexOf("http-get-map") > -1)))));
+        // let's create the buttons
+        let buttons = [];
         if (wms) {
-            return (<div className="record-buttons">
+            buttons.push(
                 <Button
-                bsStyle="success"
-                bsSize={this.props.buttonSize}
-                onClick={() => {this.addLayer(wms); }}
-                key="addlayer">
-                    <Glyphicon glyph="plus" />&nbsp;<Message msgId="catalog.addToMap" />
-                </Button>&nbsp;
-            </div>);
+                    className="record-button"
+                    bsStyle="success"
+                    bsSize={this.props.buttonSize}
+                    onClick={() => { this.addLayer(wms); }}
+                    key="addlayer">
+                        <Glyphicon glyph="plus" />&nbsp;<Message msgId="catalog.addToMap"/>
+                </Button>
+            );
         }
+        const copyToClipboardTooltip = (<Tooltip id="tooltip"><Message msgId="catalog.copyToClipboard"/></Tooltip>);
+        const copiedTooltip = (<Tooltip id="tooltip"><Message msgId="catalog.copied"/></Tooltip>);
+        let getCapabilitiesLinks = [];
+        // creating get capbilities links that will be used to share layers info
+        if (this.props.showGetCapLinks) {
+            // helper function to create a get capabilities link componeent
+            let createCapabilitiesLink = function(key, getCapabilitiesUrl, messageId, onCopy, tooltip, setLinkCopiedStatus) {
+                return (
+                    <div key={key} className="input-group get-cap-links">
+                        <input className="form-control input-sm cap-input" type="text"
+                               value={getCapabilitiesUrl} onChange={() => {}}/>
+                        <span className="input-group-btn">
+                            <CopyToClipboard text={getCapabilitiesUrl} onCopy={onCopy}>
+                                <OverlayTrigger placement="right" overlay={tooltip} onExited={() => setLinkCopiedStatus(key, false)}>
+                                    <Button bsSize="small" className="cap-button" onClick={() => setLinkCopiedStatus(key, true)}>
+                                        <Glyphicon glyph="copy"/>&nbsp;<Message msgId={messageId}/>
```

Comment:
I don't think this glyph exists in the new theme. We should include it because is useful.